tests/Mocks/SnmpQueryMock.php

```diff
@@ -73,6 +73,11 @@ public function deviceArray(array $device): SnmpQueryInterface
         return $this;
     }
 
+    public function cache(): SnmpQueryInterface
+    {
+        return $this; // ignore, always cached
```

Comment:
Never?

Reply:
It reads the whole snmprec file in to memory.